3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,196 @@
+
+//Flight booking fullname function
+
+// Define the getFullname function
+function getFullname(firstname, surname) {
+  return firstname + " " + surname;
```

Comment:
You could try to use template literals here  😉
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals